protocol/messenger_communities.go

```diff
@@ -18,6 +19,8 @@ import (
 	"github.com/status-im/status-go/protocol/transport"
 )
 
+var messageArchiveInterval = 5 * time.Minute
```

Comment:
This should be 7 * time.Day once this goes to production, but for testing a shorter interval is needed